fair_mango/dataset/dataset.py

```diff
@@ -36,19 +42,21 @@ def check_column_in_df(df: pd.DataFrame, columns: Sequence) -> None:
 def check_real_and_predicted_target_match(
     real_target: Sequence[str], predicted_target: Sequence[str]
 ) -> None:
-    """check that number of real targets and number of predicted targets match
+    """check that the number of real targets and number of predicted targets
+    match.
 
     Parameters
     ----------
     real_target : Sequence[str]
-        real targets
+        sequence of column names corresponding to the real targets
+        (true labels).
     predicted_target : Sequence[str]
-        predicted targets
+        sequence of column names corresponding to the predicted targets.
 
     Raises
     ------
     ValueError
-        if the number of targets and predicted targets does not match
+        if the number of real targets and predicted targets does not match.
     """
     if isinstance(real_target, str) and isinstance(predicted_target, str):
```

Comment:
I just tested this and providing a `str` for `Sequence[str]` is possible and it does not raise any warning. But you are right. It would be better to force it to be a `Sequence`. 

I am going to do that in the initialisation of `class Dataset` where I will force the attributes `sensitive`, `real_target`, `predicted_target` to be a `list`.